api-spec.md

```diff
@@ -125,33 +69,26 @@ Local Look-up
 #### Body Example:
 ```
 {
-  "device_id": "xxxxxx"
+  "fromDate": "timestamp",
+  "location": "city_name",
+  "getTraces": true,
```

Comment:
To avoid privacy issues, we cannot provide traces of infected person to the person at risk, so `getTraces` parameter is redundant.
Also, instead of `location` I would propose same parameter as for report submission: `"traces": [ "geo_hash1", "geo_hash2" ]`. This way server will be able to match traces of requester to traces in reports. That may be a bit computationally expensive (can be mitigate with using some algorithm like R*Tree), so for simplicity it can be just one geohash covering geographic region that includes all areas where user have been in the past X days.

Reply:
ohhh right, was thinking download paths for some reason lol,  thanks for the detailed review! @parametr